jcore-muc7-reader/src/main/java/de/julielab/jcore/reader/muc7/MUC7Reader.java

```diff
@@ -1005,13 +1007,13 @@ public void getNext(CAS cas) throws IOException, CollectionException {
 			startPosition = 0;
 			// important: keep this sequence!
 			annotateHeader(docNode);
-//			annotateSlug(docNode, textToBeProcessed);
-//			annotateDate(docNode, textToBeProcessed);
-//			annotateNumOfWords(docNode, textToBeProcessed);
-//			annotatePreamble(docNode, textToBeProcessed);
+			annotateSlug(docNode, textToBeProcessed);
+			annotateDate(docNode, textToBeProcessed);
+			annotateNumOfWords(docNode, textToBeProcessed);
+			annotatePreamble(docNode, textToBeProcessed);
 			annotateText(docNode, textToBeProcessed);
-//			annotateTrailer(docNode, textToBeProcessed);
-			annotateCorefs();
+			annotateTrailer(docNode, textToBeProcessed);
+//			annotateCorefs();
```

Comment:
Thanks for the pull request.
Why is this line uncommented? Where are the coreferences added to the CAS if not here?